styleguide/source/_patterns/02-molecules/article-metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+  "date": "Jul 28, 2017",
+  "readtime": "5 min read"
+}
```

Comment:
empty line

Reply:
I added these and github isn't picking them up. not sure why.

Reply:
It doesn't. That's fine.